xray/utils/resultstable_test.go

```diff
@@ -400,6 +401,57 @@ func TestAppendUniqueImpactPaths(t *testing.T) {
 	}
 }
 
+func TestGetSeveritiesFormat(t *testing.T) {
+	testCases := []struct {
+		input          string
+		expectedOutput string
+		expectedError  error
+	}{
+		// Test supported severity
+		{
+			input:          "critical",
+			expectedOutput: "Critical",
+			expectedError:  nil,
+		},
+		{
+			input:          "hiGH",
+			expectedOutput: "High",
+			expectedError:  nil,
+		},
+		{
+			input:          "Low",
+			expectedOutput: "Low",
+			expectedError:  nil,
+		},
+		{
+			input:          "MedIum",
+			expectedOutput: "Medium",
+			expectedError:  nil,
+		},
+		{
+			input:          "",
+			expectedOutput: "",
+			expectedError:  nil,
+		},
+		// Test unsupported severity
+		{
+			input:          "invalid_severity",
+			expectedOutput: "",
+			expectedError:  errors.New("only the following severities are supported: Critical, High, Medium and Low"),
+		},
```

Comment:
I think this is more clean
```suggestion
		// Test supported severity
		{input: "critical", expectedOutput: "Critical", expectedError: nil},
		{input: "hiGH", expectedOutput: "High", expectedError: nil},
		{input: "Low", expectedOutput: "Low", expectedError: nil},
		{input: "MedIum", expectedOutput: "Medium", expectedError: nil},
		{input: "", expectedOutput: "", expectedError: nil},
		// Test unsupported severity
		{input: "invalid_severity", expectedOutput: "", expectedError: errors.New("only the following severities are supported: Critical, High, Medium and Low")},
```

Reply:
done